src/main/java/edu/cornell/kfs/pmw/batch/service/impl/PaymentWorksBatchUtilityServiceImpl.java

```diff
@@ -70,7 +72,21 @@ public String getFileContents(String fileName) {
     }
 
     @Override 
-    public Note createNote(String noteText) {
+    public KfsVendorDataWrapper createNoteRecordingAnyErrors(KfsVendorDataWrapper kfsVendorDataWrapper, String noteText, String noteErrorDescriptor) {
+        if (noteTextSizeIsWithinLimit(noteText)) {
+            Note note = createNote(noteText);
+            kfsVendorDataWrapper.getVendorNotes().add(note);
+        } else {
+            LOG.error("createNoteRecordingAnyErrors: The " + noteErrorDescriptor + " Note contained " + noteText.length() + "characters and it can only have " + PaymentWorksConstants.NOTE_TEXT_DEFAULT_MAX_LENGTH);
+        }
+        return kfsVendorDataWrapper;
+    }
+    
+    private boolean noteTextSizeIsWithinLimit(String noteText) {
+        return (noteText.length() <= PaymentWorksConstants.NOTE_TEXT_DEFAULT_MAX_LENGTH);
```

Comment:
I am wondering if this should include a null check here as well.  StringUtils.length(noteText), will return zero if noteText is null, otherwise return the length of the string.

Reply:
Adding the null check.